draft-ietf-tls-dtls-rrc.md

```diff
@@ -52,41 +52,29 @@ protocol versions 1.2 and 1.3.
 
 # Introduction
 
-In "classical" DTLS, selecting a security context of an incoming DTLS record is
-accomplished with the help of the 5-tuple, i.e. source IP address, source port,
-transport protocol, destination IP address, and destination port.  Changes to
-this 5 tuple can happen for a variety reasons over the lifetime of the DTLS
-session.  In the IoT context, NAT rebinding is common with sleepy devices.
-Other examples include end host mobility and multi-homing.  Without CID, if the
-source IP address and/or source port changes during the lifetime of an ongoing
-DTLS session then the receiver will be unable to locate the correct security
-context.  As a result, the DTLS handshake has to be re-run.  Of course, it is
-not necessary to re-run the full handshake if session resumption is supported
-and negotiated.
-
 A CID is an identifier carried in the record layer header of a DTLS datagram
 that gives the receiver additional information for selecting the appropriate
 security context.  The CID mechanism has been specified in {{!RFC9146}} for
 DTLS 1.2 and in {{!RFC9147}} for DTLS 1.3.
 
 Section 6 of {{!RFC9146}} describes how the use of CID increases the attack
-surface by providing both on-path and off-path attackers an opportunity for
+surface of DTLS 1.2 and 1.3 by providing both on-path and off-path attackers an opportunity for
 (D)DoS.  It then goes on describing the steps a DTLS principal must take when a
 record with a CID is received that has a source address (and/or port) different
 from the one currently associated with the DTLS connection.  However, the
 actual mechanism for ensuring that the new peer address is willing to receive
-and process DTLS records is left open.  This document standardizes a return
-routability check (RRC) as part of the DTLS protocol itself.
+and process DTLS records is left open.  To address the gap, this document defines a return
+routability check (RRC) sub-protocol for DTLS 1.2 and 1.3.
 
-The return routability check is performed by the receiving peer before the
-CID-to-IP address/port binding is updated in that peer's session state
-database.  This is done in order to provide more confidence to the receiving
-peer that the sending peer is reachable at the indicated address and port.
+The return routability check is performed by the receiving endpoint before the
+CID-address binding is updated in that endpoint's session state.
+This is done in order to give the receiving endpoint confidence
+that the sending peer is in fact reachable at the latest indicated address and port.
```

Comment:
Assuming "multiple address changes" and potential "multiple RRC", the phrase "latest indicated address and port" doesn't fit. Isn't it the "tested" one? Or something in that direction.

Reply:
I think in this case _indicated_ is not wrong (maybe there's a more fitting English word but my mastery of the language is limited.)

The wannabe sending peer (or some other box on the path) has _indicated_ its transport address in the UDP (or DCCP?) / IP header, which RRC needs to _test_ to confirm (or deny) actual reachability.

Reply:
As per f2f discussion: clarified in [a69e9a2](https://github.com/tlswg/dtls-rrc/pull/63/commits/a69e9a2f49c884074bfa0de4d7097e90e7b729ac)